virttest/libvirt_xml/vm_xml.py

```diff
@@ -2163,6 +2164,11 @@ def __init__(self, virsh_instance=base.virsh):
                                parent_xpath='/',
                                tag_name='hpt',
                                attribute='resizing')
+        accessors.XMLAttribute(property_name='htm',
```

Comment:
I think it is okay to merge this patch as framework would raise `ValueError` if unsupported libvirt is used with htm. From framework perspective it is good to have this enablement.